packages/react-components/src/components/Announcer/Announcer.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
The reason why we have these files is because it's not exportable from northstar, I guess?

Reply:
See here: https://github.com/Azure/communication-ui-library/pull/3005. The component we used has react 16 as peer dependency but it is a tiny package so we are just taking the code from it directly